### DISCUSSION TITLE
Auto display on PrimeNg megamenu

### DISCUSSION THREAD

**Discussion by thomasNDS** [primefaces/community#608] [PrimeNG] [answered]
On Megamenu component, add the same options of menubar component  https://primeng.org/menubar autoDisplay, autoHide and autoHideDelay to auto display menus on hover.

It's also exist on Primefaces JSF Megamenu component https://primefaces.github.io/primefaces/12_0_0/#/components/megamenu

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, autoDisplay is default behavior of the megamenu component. For the rest, we've added them to our internal backlog.
